### ISSUE TITLE
(python3) Installing same python version 32bits and 64bits variant, possible?

### ISSUE THREAD

**Issue by FrancescElies** [chocolatey-community/chocolatey-coreteampackages#1485]
## Context
I am interested in installing python 32bits and 64bits with debugging symbols via chocolatey.

## Current Behavior
So far, if I do the following I can manage to install what I want, for the sake of an example let's have version 3.7.0.

```
choco install python3 --forcex86 --allowmultiple --version=3.7.0  --installargs "TargetDir=C:\Python37_32bits CompileAll=1 Include_debug=1 Include_symbols=1"
choco install python3 --force    --allowmultiple --version=3.7.0  --installargs "TargetDir=C:\Python37        CompileAll=1 Include_debug=1 Include_symbols=1"
```
At the moment I need `--forcex86` to tell choco toinstall python-32bits version. Later if I want to install the same python version but the 64bits variante I need `--force` to tell chocolatey, please don't complain you already installed that version.

**The problem:**

Afterwards chocolatey seems not to know he has installed two versions of the same package.

Is there any clean way to achieve installing pythonXYZ 32bits and 64bits and manage them via chocolatey?

I guess that if 64bits and 32 bits would have been diffrerent packages chocolatey would not have problems with it. Related to #696.

## Expected Behavior
I would like  chocolatey to be able to install python x.y.z 32bits and 64bits on the same machine without running into conflicts.

## Your Environment
* Package Version used: x.y.x
* Operating System and version: Microsoft Windows 10 Enterprise
* Chocolatey version: 0.10.15

**Comment by AdmiringWorm** [comment]
> Afterwards chocolatey seems not to know he has installed two versions of the same package.

I assume that could be because it is the same version that got installed.
When you call the installation again in your second call, you are actually replacing the first installation instead of installing a new version (would have been different if the second call was a different version). That is just an assumption on my part.

> I guess that if 64bits and 32 bits would have been diffrerent packages chocolatey would not have problems with it. Related to #696.

While it is true that having different packages for each architecture would allow them to be installed side-by-side, that isn't a solution IMO.

I think something different should be used. If we add a package parameter that allows installing both 64bit and 32bit at the same time, would that be something that could work for you?

**Comment by FrancescElies** [comment]
> I think something different should be used. If we add a package parameter that allows installing both 64bit and 32bit at the same time, would that be something that could work for you?

That would work for me, I am mainly interested in this being possible. If you the maintainers are fine with that I'm more than happy.

How do we move forward now?

**Comment by AdmiringWorm** [comment]
> How do we move forward now?

Well, we need to first decide on which parameters should be used to allow the installation of both 64bit and 32bit.
I am thinking a couple of options in this regard:

1. Have a parameter called `/Include32Bit` that will install the 32bit into a default installation directory (only available on 64bit systems, of course).
2. Have a parameter called `/InstallDir32:path` that allows a custom path for the 32bit directory.

That is the main thing to decide upon, the second is to get someone that has the time and ability to implement the necessary logic for such an implementation. I could look into getting that performed, but that won't happen until next week at the earliest (but could take up to a whole month, not entirely sure yet).

**Comment by FrancescElies** [comment]
Hey tanks for your reply, `/InstallDir32:path` sounds good to me.
I will check internally how pressing is this one for us, is this something someone without knowing the repo could tackle?

**Comment by AdmiringWorm** [comment]
> I will check internally how pressing is this one for us, is this something someone without knowing the repo could tackle?

I believe someone without knowledge about the repo can tackle this.
It will require at least some fundamental knowledge of chocolatey packages and its different helpers.

**Comment by AdmiringWorm** [comment]
This request have now been implemented in the python3 package, and a package fix for version 3.8.x is in the process of getting submitted to chocolatey.org (fix versions for older versions and beta versions are not planned at this time).

**Comment by FrancescElies** [comment]
Where could I followup the submit process?

**Comment by AdmiringWorm** [comment]
@FrancescElies I call it a process, but honestly it is just one of us forcing a new update of a package on appveyor.

The latest changes are available: https://chocolatey.org/packages/python/3.8.5.20200721

**Comment by FrancescElies** [comment]
I gave this one a try as follows
`choco install python3     --allowmultiple --version=3.8.5  --installargs "CompileAll=1 Include_debug=1 Include_symbols=1" --params "/InstallDir32:C:\Python38_32bits" -y`
`InstallDir32` was ignored and installed under C:\Python38
![image](https://user-images.githubusercontent.com/6558089/90126914-ae5cb200-dd64-11ea-88c7-19b6964fa847.png)

Did I do something wrong?

**Comment by AdmiringWorm** [comment]
@FrancescElies you tried it on the wrong version. You need to use the fix version that was pushed: `3.8.5.20200721`.
Existing versions can't be updated due to them being immutable on chocolatey.org
